libcodechecker/source_code_comment_handler.py

```diff
@@ -193,8 +193,13 @@ def get_source_line_comments(self, bug_line):
             if has_any_marker:
                 rev = list(reversed(curr_suppress_comment))
                 suppress_comment = ''.join(rev).replace('//', '')
-                source_line_comments.append(
-                    self.__process_source_line_comment(suppress_comment))
+                comment = self.__process_source_line_comment(suppress_comment)
+                if comment:
+                    source_line_comments.append(comment)
+                else:
+                    LOG.warning("Misspelled review status comment: %s",
```

Comment:
One more thing, otherwise LGTM. Can we write out the file path and the line number too? It would be easier to understand where the user can find the problem.